jdaviz/components/tray_plugin.vue

```diff
@@ -45,6 +45,10 @@ module.exports = {
       if (!document.hidden) {
         this.$emit('plugin-ping', Date.now())
       }
+      if (this.scroll_to) {
+        this.$emit('update:scroll_to', false)
+        this.$el.scrollIntoView({behavior: "smooth", block: "start"});
+      }
```

Comment:
https://github.com/spacetelescope/jdaviz/assets/877591/77ff2c2b-36f3-4920-91f4-dde479aee820

Reply:
ah `block: "nearest"` helps, but then scrolls to the bottom of the plugin when it needs to scroll up instead of down 🤔

Reply:
I can reproduce that behaviour in Firefox, chromium doens't have this.

This would fix it, although it might be a bit brittle to rely on the `.overflow-y-auto` class, which might be susceptible to changes without consideration for its usage here: 
```javascript
const viewport = this.$el.closest(".overflow-y-auto")
const scrollTarget = this.$el.closest(".v-expansion-panel")
viewport.scrollTo({top: scrollTarget.offsetTop, behavior: "smooth"})
// this.$el.scrollIntoView({behavior: "smooth", block: "nearest", inline: "start"});
```